doc/nrf/releases/release-notes-latest.rst

```diff
@@ -52,6 +52,8 @@ nRF9160
 
     * Updated to prevent reinitialization of param list in :c:func:`modem_info_init`.
 
+  * :ref:`lib_fota_download` library - Added an API to get image type that is being downloaded.
```

Comment:
```suggestion
  * :ref:`lib_fota_download` library:
  
     * Added an API to retrieve the image type that is being downloaded.
```

Reply:
Updated, thank you. More updates will be done to this library